.github/workflows/test-unit-snapshot.yml

```
@@ -1,4 +1,4 @@
-name: Test - Unit and Snapshot
+name: Test - Unit, Snapshot
```

Let's not rename the workflows unless it's a meaningful change, otherwise it makes the github actions list messy
```suggestion
name: Test - Unit and Snapshot
```

---

This comes probably from one of my PRs. I did rename it there to: `Test - Unit, Integration, Snapshot`, as there are tests for the middleware as well.. Probably just not reverted properly here.

---

Yes, the last change was `Test - Unit, Integration, Snapshot` and I simply removed the word `Integration, `.
Changing it back to `Unit and Snapshot` right away.